### PR TITLE
Added some setup code for backend

### PR DESCRIPTION
**Pull request by MerryRosalie**

### CODE REVIEW

File: backend/src/routes/notesRoutes.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import express, { Request, Response } from "express";
+
+const router = express.Router();
+
+router.get("/", (req: Request, res: Response) => {
+  res.send("i love boobs");
```
Comment (floyrin) [inline backend/src/routes/notesRoutes.ts]: 💀 no please

File: backend/src/server.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+const app = express();
+app.use(express.json());
+// Use middleware that allows for access from other domains
+app.use(cors());
+// for logging errors
+app.use(morgan("dev"));
+
+app.get("/", (req: Request, res: Response) => {
+  res.send("ur mom");
```
Comment (floyrin) [inline backend/src/server.ts]: 💀💀💀💀💀💀💀💀💀💀💀

**Review by floyrin — CHANGES_REQUESTED** [review]

File: backend/src/routes/notesRoutes.ts (hunk 1 above)
Comment (MerryRosalie) [reply backend/src/routes/notesRoutes.ts]: b-but 🥺

**Review by floyrin — APPROVED** [review]

**Comment by floyrin** [comment]
yes

**Pull request merged**